virtme/guest/virtme-sshd-script

```diff
@@ -24,18 +24,37 @@ SSH_AUTH_KEYS="${SSH_HOME}/.ssh/authorized_keys"
 if [ "$(stat -f -c "%t" "${SSH_AUTH_KEYS}")" = "${OVERLAYFS}" ]; then
     cat "${SSH_HOME}"/.ssh/id_*.pub >> "${SSH_AUTH_KEYS}" 2>/dev/null
     chown "${virtme_ssh_user}" "${SSH_AUTH_KEYS}" 2>/dev/null
+    chmod 600 "${SSH_HOME}/.ssh/authorized_keys" 2>/dev/null
 fi
 
 # Generate ssh host keys (if they don't exist already).
 CACHE_DIR=${SSH_HOME}/.cache/virtme-ng/.ssh
 mkdir -p "${CACHE_DIR}/etc/ssh"
 ssh-keygen -A -f "${CACHE_DIR}"
+
+# Generate a minimal sshd config.
+SSH_CONFIG=/etc/ssh/sshd_config
+if [ "$(stat -f -c "%t" "${SSH_CONFIG}")" = "${OVERLAYFS}" ]; then
+    ssh_dir=$(dirname "${SSH_CONFIG}")
+    mkdir -p "${ssh_dir}"
+    cat << EOF > "${SSH_CONFIG}"
+# This file is automatically generated by virtme-ng.
+Port 22
+PermitRootLogin yes
+AuthorizedKeysFile .ssh/authorized_keys
+PubkeyAuthentication yes
+UsePAM yes
+PrintMotd no
+EOF
```

Comment:
detail: you can also use `-EOF` to keep the indentation after the `if` statement

```suggestion
    cat << -EOF > "${SSH_CONFIG}"
        # This file is automatically generated by virtme-ng.
        Port 22
        PermitRootLogin yes
        AuthorizedKeysFile .ssh/authorized_keys
        PubkeyAuthentication yes
        UsePAM yes
        PrintMotd no
    EOF
```

Reply:
oh nice! TIL

Reply:
Hm... this doesn't seem to work for me. Is -EOF only removing tabs maybe?

Reply:
> Hm... this doesn't seem to work for me. Is -EOF only removing tabs maybe?

oops yes, sorry, I'm usually using tabs :)

(also there should not be a space before the `-EOF`)

Reply:
A workaround is to use:

```
  printf "%s\n" \
      "line 1" \
      "line 2" \
      ...
```

or `sed`. But not as well integrated than this `cat <<-EOF` :)